src/Prover-Axelar.sol

```diff
@@ -15,14 +15,14 @@ import { Misc } from "lib/filecoin-solidity/contracts/v0.8/utils/Misc.sol";
 import { FilAddresses } from "lib/filecoin-solidity/contracts/v0.8/utils/FilAddresses.sol";
 import { DataAttestation, IBridgeContract, StringsEqual } from "./Oracles.sol";
 import {Strings} from "lib/openzeppelin-contracts/contracts/utils/Strings.sol";
+import {Ownable} from "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
 import {AxelarExecutable} from "lib/axelar-gmp-sdk-solidity/contracts/executable/AxelarExecutable.sol";
 import { IAxelarGateway } from 'lib/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol';
 import { IAxelarGasService } from 'lib/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGasService.sol';
 
-
 using CBOR for CBOR.CBORBuffer;
 
-contract DealClient is AxelarExecutable {
+contract DealClient is AxelarExecutable, Ownable {
```

Comment:
Id kind of rather we just left this unowned either hardcoded the map with a few values in the contructor or did a write once thing as we do in setting the oracle.  My thinking is that all off chain state is a pain to manage and adding an owner adds to that state.

Reply:
agreed